arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -937,6 +937,7 @@
         "TTL": {
           "type": "integer",
           "format": "int64",
+	   "x-ms-client-name": "ttl",
```

Comment:
@MikhailTryakhov : Have you generated the code to test this works? @amarzavery and I looked at the code that would get generated and it looks like in case of `TTL` or `ttl` the code that gets generated is `Ttl`